### ISSUE TITLE
some text on the page is not readable with screen readers

### ISSUE THREAD

**Issue by musicalman** [Troxi54/Points-Progression#3]
Hi,
I am visually impaired and use a screen reader (in my case NVDA). I have been playing points Progression for a few days now, and while it is playable with a screen reader and keyboard, I cannot read some of the helpful information that seems to be near the progress bars, such as the goal, or effect of fastest run. I only know this stuff is shown because I ran OCR on the window, and it picked up the text. But since it's trying to extract text from an image, and not reading the text natively, it makes mistakes such as turning numbers into letters. Therefore I hope there is a way to properly expose this text to screen readers so it can be read properly. I hope I'm making sense, I know nothing about game/web development.

I'm still going to continue playing though, since it looks like the text I can't read is more helpful than vital. Just makes strategizing a bit harder. I'm not that far into the game, I've just purchased tier upgrade 2, so I have no idea how the experience will be going forward.

If you have any thoughts or questions, let me know! I'll also try some more stuff on my end to see if I can figure something out.
